docs/sphinx/examples/cpp/algorithms/grover.cpp

```diff
@@ -3,44 +3,61 @@
 // nvq++ grover.cpp -o grover.x && ./grover.x
 // ```
 
+#include <cmath>
 #include <cudaq.h>
+#include <numbers>
 
-__qpu__ void reflect_about_uniform(cudaq::qview<> q) {
-  auto ctrlQubits = q.front(q.size() - 1);
-  auto &lastQubit = q.back();
+__qpu__ void reflect_about_uniform(cudaq::qvector<> &qs) {
+  auto ctrlQubits = qs.front(qs.size() - 1);
+  auto &lastQubit = qs.back();
```

Comment:
I think these minor tweaks to examples should be in a different PR.

Reply:
Agreed. Will move it out. The plain wrong example just bothered me.

Reply:
https://github.com/bettinaheim/cuda-quantum/pull/21